### PR TITLE
Streaming support

### PR DESCRIPTION
**Pull request by bartdevylder**

### CODE REVIEW

**Comment by bartdevylder** [comment]
Switching between the two versions (streaming_enabled true/false) was tested on qa11045

File: connect.sh [hunk 1]
```diff
@@ -29,13 +29,27 @@ else
     LOG=/var/log/rabbitmq/rabbit*.log
   elif [ "$SERVICE" == "rum" ]; then
     LOG=/var/log/nginx/*.log
+  elif [ "$SERVICE" == "streamingroller" ]; then
+    PURE_DOCKER=true
```
Comment (fryckbos) [inline connect.sh]: PURE_DOCKER -> DOCKER_LOGS

File: misc/kafka [hunk 2]
```diff
@@ -0,0 +1 @@
+-e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://:9092
```
Comment (fryckbos) [inline misc/kafka]: Should we add this to our Kafka dockerfile ?

File: misc/zookeeper [hunk 3]
```diff
@@ -0,0 +1 @@
+-e ZOOKEEPER_CLIENT_PORT=32181 -e KAFKA_JMX_PORT=9997 -e KAFKA_JMX_HOST=localhost
```
Comment (fryckbos) [inline misc/zookeeper]: Should we add this to our Zookeeper dockerfile ?

File: scripts/consumer-group-seek-to-end.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+./connect.sh kafka JMX_PORT=1234 kafka-consumer-groups --bootstrap-server localhost:9092 --reset-offsets --to-latest --execute --all-topics --group $1
```
Comment (fryckbos) [inline scripts/consumer-group-seek-to-end.sh]: What's the reason for the JMX_PORT ?

File: scripts/describe-topic.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+./connect.sh kafka kafka-topics --zookeeper kafka3:2181 --describe --topic $1
```
Comment (fryckbos) [inline scripts/describe-topic.sh]: kafka3:2181 -> zookeeper:32181

File: scripts/list-topics.sh [hunk 6]
```diff
@@ -0,0 +1 @@
+./connect.sh kafka kafka-topics --zookeeper kafka3:2181 --list
```
Comment (fryckbos) [inline scripts/list-topics.sh]: kafka3:2181 -> zookeeper:32181

File: scripts/manage-topics.sh [hunk 7]
```diff
@@ -0,0 +1,2 @@
+. conf.sh
+docker run -it --rm coscale/streamingroller:$VERSION java -cp /opt/coscale/streamingroller/bin/streamingroller-$VERSION-jar-with-dependencies.jar coscale.streamingcore.kafka.ManageTopics -h kafka1:9092 $@
```
Comment (fryckbos) [inline scripts/manage-topics.sh]: Add link + kafka1 -> kafka ?

**Review by fryckbos — COMMENTED** [review]
Some small fixes. I'm also thinking we should perhaps group the kafka scripts in ./diagnose.sh ?

**Review by fryckbos — APPROVED** [review]
Looks good to me.

**Pull request merged**